Library/Homebrew/test/cask/audit_spec.rb

```diff
@@ -399,6 +399,20 @@ def include_msg?(messages, msg)
       end
     end
 
+    describe "blacklist checks" do
+      context "when the Cask isn't blacklisted" do
+        let(:cask_token) { "local-transmission" }
```

Comment:
@reitermarkus I tried to use `local-caffeine` but the audit failed the stanza check as there isn't a `name` defined on that test Cask. Will `local-transmission` work for the audit pass test?